### PR TITLE
chore: implement music and user objects

### PR DESCRIPTION
**Pull request by biancasoliz-uwu**
## Description

Provide a small description of Pull Request.

## Type of Change

Put an `x` in all the boxes that apply:

- [x] ✨ New Feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug Fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking Change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code Refactor
- [ ] ✅ Build Configuration Change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Pull Request Checklist

Put an `x` in all the boxes that apply:

- [ ] My commit messages are detailed.
- [x] My code follows the code style of this project.
- [ ] No existing features have been broken without good reason.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: src/main/java/org/example/objects/Music.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        this.popularity = popularity;
+        this.danceability = danceability;
+        this.energy = energy;
+        this.speechiness = speechiness;
+        this.acousticness = acousticness;
+        this.instrumentalness = instrumentalness;
+        this.liveness = liveness;
+        this.valence = valence;
+        this.trackPreviewUrl = trackPreviewUrl;
+    }
```
Comment (teran-joseluis) [inline src/main/java/org/example/objects/Music.java]: @biancasoliz-uwu Do not forget to add the necessary methods to access the attributes.

File: src/main/java/org/example/objects/User.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    public User(int id, String name, int age, List<String> favoriteGenres, List<String> favoriteArtists,
+                   List<String> favoriteSongs, List<String> recentlyPlayed) {
+        this.id = id;
+        this.name = name;
+        this.age = age;
+        this.favoriteGenres = favoriteGenres;
+        this.favoriteArtists = favoriteArtists;
+        this.favoriteSongs = favoriteSongs;
+        this.recentlyPlayed = recentlyPlayed;
+    }
```
Comment (teran-joseluis) [inline src/main/java/org/example/objects/User.java]: @biancasoliz-uwu Do not forget to add the necessary methods to access the attributes.

File: src/main/java/org/example/objects/Music.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+public class Music {
+    private int id;
+    private String trackName;
+    private String artistNames;
+    private String albumName;
+    private String albumArtistNames;
+    private String albumReleaseDate;
+    private String albumImageUrl;
+    private String artistGenres;
```
Comment (Misax148) [inline src/main/java/org/example/objects/Music.java:11]: artistGenres we will manage as a ArrayList, please change de data type.

File: src/main/java/org/example/objects/Music.java (hunk 1 above)
Comment (teran-joseluis) [reply src/main/java/org/example/objects/Music.java]: @biancasoliz-uwu I'm sorry to tell you but for the progress of the project I had to implement the methods that we need to start working.

File: src/main/java/org/example/objects/User.java (hunk 2 above)
Comment (teran-joseluis) [reply src/main/java/org/example/objects/User.java]: @biancasoliz-uwu I'm sorry to tell you but for the progress of the project I had to implement the methods that we need to start working.

**Review by teran-joseluis — APPROVED** [review]

**Review by Misax148 — APPROVED** [review]

**Pull request merged**